dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/InstallFeatureTest.java

```diff
@@ -929,4 +950,122 @@ public void testComplexVersionTolerates() throws Exception {
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "proxyPort", proxyPort);
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "proxyUser", "wasngi");
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "proxyPassword", "test");
+	    
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "mavenCentralMirror.url", nexusURL);
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "mavenCentralMirror.user", "admin");
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "mavenCentralMirror.password", "golf");
+	    
+	    String[] param1s = { "installFeature", "json-1.0", "--verbose" };
+	    String[] filesList = { "/lib/features/com.ibm.websphere.appserver.json-1.0.mf" };
+	    ProgramOutput po = runFeatureUtility(METHOD_NAME, param1s);
+
+	    try {
+		checkCommandOutput(po, 0, null, filesList);
+	    } catch (Exception e) {
+		  ExecResult lsResult = proxyContainer.execInContainer("cat", "/var/log/squid/access.log");
+		  String stdout = lsResult.getStdout();
+		  Log.info(c, METHOD_NAME, "Test Failed. Proxy Log: " + stdout);
+	    }
+	    
+	    Log.exiting(c, METHOD_NAME);
+
+	}
+
+	@Test
+	public void testProxyAndRepoAuthEncoded() throws Exception {
+	    final String METHOD_NAME = "testProxyAndRepoAuthEncoded";
+	    Log.entering(c, METHOD_NAME);
+
+	    container.dependsOn(nexusContainer).start();
+
+	    String proxyHost = "http://" + proxyContainer.getHost();
+	    String proxyPort = proxyContainer.getMappedPort(3128).toString();
+	    String nexusURL = "http://nexus:8081/repository/maven-central/";
+
+	    // overwrite the local maven repo so it can fetch the artifact from nexus repo
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "featureLocalRepo", "tmp");
+
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "proxyHost", proxyHost);
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "proxyPort", proxyPort);
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "proxyUser", "wasngi");
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "proxyPassword", "{xor}KzosKw==");
+
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "mavenCentralMirror.url", nexusURL);
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "mavenCentralMirror.user", "admin");
+	    writeToProps(minifiedRoot + "/etc/featureUtility.properties", "mavenCentralMirror.password",
+		    "{xor}ODAzOQ==");
+
+	    String[] param1s = { "installFeature", "json-1.0", "--verbose" };
+	    String[] filesList = { "/lib/features/com.ibm.websphere.appserver.json-1.0.mf" };
+	    ProgramOutput po = runFeatureUtility(METHOD_NAME, param1s);
+
+	    try {
+		checkCommandOutput(po, 0, null, filesList);
+	    } catch (Exception e) {
+		ExecResult lsResult = proxyContainer.execInContainer("cat", "/var/log/squid/access.log");
```

Comment:
This code seems to be reused repeatedly, maybe create a method and add the exception tracing i've suggested in there